portal/cookbook-tasklist.md

```diff
@@ -0,0 +1,51 @@
+# Cookbook Author Checklist Template
+
+Project Pythia maintainers will provide this checklist to Cookbook authors to help get Cookbooks ready for publication.
+
+```md
+**I am your Cookbook advocate, and my GitHub handle is <my-username>. Please tag me in this issue with any problems getting your Cookbook published!**
+Once we've marked this entire checklist, [click here to open an issue on ProjectPythia/cookbook-gallery to publish your Cookbook!](https://github.com/ProjectPythia/cookbook-gallery/issues/new?assignees=ProjectPythia%2Feducation&labels=content%2Ccookbook-gallery-submission&projects=&template=update-cookbook-gallery.yaml&title=Update+Gallery+with+new+Cookbook)
```

Comment:
My thought was that these are created as soon as an "advocate" is assigned to the Cookbook, and can help guide development such that when someone clicks "Submit a new Cookbook", they really are ready to submit.

I'd bet refactoring our Cookbook guide to be more of a checklist and then having this be more trim and only opened after submission would accomplish the same goal, if we prefer that approach.

Reply:
Sorry, I must be missing something. What triggers the assignment of an advocate? I thought this happened when the author clicks "submit a new cookbook"

Reply:
It looks like that is done manually? I don't see an action that would do this.

Reply:
How does anyone know to do it manually? Is the Pythia team even aware of the cookbook under development before the creator clicks "submit a new cookbook"?

Reply:
The "Submit a New Cookbook" link is for filling out our issue template on the gallery. If the machinery is still working, that issue kicks off our automation to open a PR to update the gallery. We placed this button at the end of the Cookbook Guide because historically we tried to guide folks through the full creation process before requesting publication to the gallery.

We don't currently have any automation for assigning advocates to Cookbooks. This has all been manual coming out of hackathon interactions, Discourse discussion, or personal communication, and we have no dedicated process for these assignments or tracking them. We are usually made fully aware of Cookbook developments via Discourse posts or by the time someone is looking to transfer a repo to the Project Pythia org, per the Cookbook Guide instructions.

We can inject this checklist whenever and wherever it's more useful, and I'll make the specifics of the wording match. Then we should build out the advocate and reviewer process structure and tooling from there. For now the primary goal I have with this PR is to get currently languishing Cookbooks cared for and to be step 1 towards the broader review process.

Reply:
I added some suggested text above advising folks to contact the Pythia team to be assigned an Advocate and initiate the review process, if they haven't already. I think this should typically happen *before* opening the PR to include the cookbook on the gallery, though it shouldn't break anything if someone pushes the button and opens the PR earlier in the process.

I agree with @dcamron that we should view this as a step toward a more formal review process, and the primary audience right now is the group of people who participated in the 2024 hackathon.

At our current stage, I don't think we should be trying to build automation around review or advocate assignments. We can handle these assignments manually easily enough. I think we need to prioritize giving people the information they need to successfully publish their cookbooks.